website/src/pages/en/resources/claud-mcp.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Claude MCP
+---
+
+This guide walks you through configuring Claude Desktop to use The Graph ecosystem's Model Context Protocol (MCP) resources: Token API and Subgraph. These integrations allow you to interact with blockchain data through natural language conversations with Claude.
+
+## What You Can Do
+
+With these integrations, you can:
+
+- **Token API**: Access token and wallet information across multiple blockchains
+- **Subgraph**: Find relevant Subgraphs for specific keywords and contracts, explore Subgraph schemas, and execute GraphQL queries
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/) installed and available in your path
+- [Claude Desktop](https://claude.ai/download) installed
+- API keys:
+  - Token API key from [The Graph Market](https://thegraph.market/)
+  - Gateway API key from [Subgraph Studio](https://thegraph.com/studio/apikeys/)
+
+## Configuration Steps
+
+### 1. Open Configuration File
+
+Navigate to your `claude_desktop_config.json` file:
+
+> **Claude Desktop** > **Settings** > **Developer** > **Edit Config**
+
+Paths by operating system:
+
+- OSX: `~/Library/Application Support/Claude/claude_desktop_config.json`
+- Windows: `%APPDATA%\Claude\claude_desktop_config.json`
+- Linux: `.config/Claude/claude_desktop_config.json`
+
+### 2. Add Configuration
+
+Replace the existing configuration with this template:
+
+```json
+{
+  "mcpServers": {
+    "subgraph": {
+      "command": "npx",
+      "args": ["mcp-remote", "--header", "Authorization:${AUTH_HEADER}", "https://subgraphs.mcp.thegraph.com/sse"],
+      "env": {
+        "AUTH_HEADER": "Bearer GATEWAY_API_KEY"
+      }
+    }
+  }
+}
+```
+
+### 3. Add Your API Keys
+
+1. Replace `ACCESS_TOKEN` with your Token API key from [The Graph Market](https://thegraph.market/)
+2. Replace `GATEWAY_API_KEY` with your Gateway API key from [Subgraph Studio](https://thegraph.com/studio/apikeys/)
+
+### 4. Save and Restart
+
+Save the configuration file and restart Claude Desktop.
+
+## Using The Graph Resources in Claude
+
+After configuration:
+
+1. Start a new conversation in Claude Desktop
+2. Click on the context menu (top right)
+3. Add "The Graph" resource by entering `graphql://subgraph` for Subgraph MCP
+
+**Important**: You must manually add The Graph resources to your chat context for each conversation where you want to use them.
+
+## Example Queries
+
+Here are some example queries you can try after setting up the resources:
+
+### Subgraph Queries
+
+```
+What are the top pools in Uniswap?
+```
```

Comment:
Interesting to use code blocks for these. I guess it's to make them easy to copy?

Reply:
Yes, to make it easier to copy